dotcom-rendering/docs/contributing/how-to.md

```diff
@@ -25,3 +37,41 @@ If you need JavaScript on the client, there are two main options:
 
 Both of these approaches should be used with caution, though. Feel free to reach
 out to the team if you're working on a feature in DCR which needs client-side scripts!
+
+## How can I create an 'island' in DCR?
+
+The 'islands' pattern is DCR's way of adding client-side React code to the site.
+You can read more about it in our
+[Improved Partial Hydration](architecture/027-better-partial-hydration.md)
+decision document.
+
+To add an island:
+
+1. Wrap your component on the server with an `<Island>` component.
+2. Add `.importable` to the component filename. Eg: `[MyThing].importable.tsx`
+3. Specify what should trigger hydration (e.g. waiting until the component
+  scrolls into view). See `Island.tsx` props for options.
+
+## How can I make an AJAX request in DCR?
+
+DCR doesn't make calls to external services from the server. The server's job is only to
+render the data passed to it by a POST request. But it does make requests on the client side.
+
+We currently use [SWR](https://swr.vercel.app/) to manage AJAX requests on the
+client. Your starting point should be the [`useApi` hook](../../src/web/lib/useApi.tsx),
+which is DCR's wrapper around SWR's own hook. Because this hook requires
+client-side React code, you will need to place your component in an `<Island>` wrapper.
+
+## What should I do if Chromatic flags up changes which are unrelated to my PR?
```

Comment:
Should this maybe live in the PR template as it is likely to become redundant and might just get stale here?

Reply:
this sounds like a great idea 👍